lightning/src/ln/channelmanager.rs

```diff
@@ -11945,11 +11915,10 @@ where
 		(self.highest_seen_timestamp.load(Ordering::Acquire) as u32).write(writer)?;
 		(self.highest_seen_timestamp.load(Ordering::Acquire) as u32).write(writer)?;
 
-		(pending_inbound_payments.len() as u64).write(writer)?;
-		for (hash, pending_payment) in pending_inbound_payments.iter() {
-			hash.write(writer)?;
-			pending_payment.write(writer)?;
-		}
+		// LDK versions prior to 0.0.104 wrote `pending_inbound_payments` here, with deprecated support
```

Comment:
```suggestion
		// LDK versions prior to 0.0.104 wrote `pending_inbound_payments` here, with deprecated support
```
if the version number right? probably is `0.0.115` but I can miss understanding what you want to say

Reply:
Hmm, why do you think it's 115?